cfgov/v1/util/util.py

```diff
@@ -33,6 +33,8 @@ def get_form_id(page):
     else:
         return 0
 
+def slugify(string):
```

Comment:
Django has a [django.utils.text.slugify](https://docs.djangoproject.com/en/1.8/ref/utils/) function built-in.